app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
Sorry, I somehow missed your answer until just now. 

![aligned](https://cloud.githubusercontent.com/assets/1538146/15656836/be50b234-26dd-11e6-9207-18b32f42c92c.png)
I wanted to have it aligned and did it almost as you suggested. 
By doing that I had a tinker at the original design. Please take a look at that, I'm pretty sure you can think of a better solution. 
I want to have the created-col left-aligned to the author, on the other hand I want the spacer to take up as much space as possible. On the other hand, I do want to prevent unnecessary word-wraps within the sentences' titles. 
All of that does currently work but I changed the original design to achieve that.